tensorflow-horovod-dist-train.md

```diff
@@ -0,0 +1,112 @@
+# Distributed training using TensorFlow and Horovod on Amazon EKS 
+
+This document explains how to perform distributed training on [Amazon EKS](https://aws.amazon.com/eks/) using TensorFlow and Horovod with synthetic dataset. 
+
+##Prerequisite
+  A. EKS setup with worker nodes
```

Comment:
Add link to EKS setup instructions? https://github.com/arun-gupta/k8s-machine-learning/blob/master/eks-gpu.md

Reply:
+1